src/uct/sm/mm/base/mm_ep.c

```diff
@@ -228,11 +230,11 @@ static inline void uct_mm_ep_update_cached_tail(uct_mm_ep_t *ep)
  * is_short = 1 - perform AM short sending
  * is_short = 0 - perform AM bcopy sending
  */
-static UCS_F_ALWAYS_INLINE ssize_t
-uct_mm_ep_am_common_send(uct_mm_send_op_t send_op, uct_mm_ep_t *ep,
-                         uct_mm_iface_t *iface, uint8_t am_id, size_t length,
-                         uint64_t header, const void *payload,
-                         uct_pack_callback_t pack_cb, void *arg)
+static UCS_F_ALWAYS_INLINE ssize_t uct_mm_ep_am_common_send(
```

Comment:
please update comment to this func, which does not seem relevant now

Reply:
The lines looks redundant since enum is used instead of integer values. So I've remove the obsolete lines from the comment.